src/dsi_milestone_2.md

```diff
@@ -9,9 +9,12 @@ rights: Creative Commons - BY-NC
 ---
 
 # Modelling
-## Plan and Execution
 
-For this part of the design process we gathered all the artifacts generated in the Research phase and tried to create meaningful groups of information. For that, we chose the bottom-up approach, since it was the most convenient for a distributed online workflow. In addition to that, our data was already skewed towards certain trends and populations (Undergraduate male) and therefore we thought that it wouldn’t be hard to identify clusters of behavioral variables.
+## Introduction
+
+For this part of the design process we have gathered all the artifacts generated in the Research phase, and have tried to create meaningful groups of information. For that, we have chosen the bottom-up approach, since it was the most convenient for a distributed online workflow. In addition to that, our data is already skewed towards certain trends and populations (Undergraduate male), and therefore we think that it won’t be hard to identify clusters of behavioral variables.
```

Comment:
Bien. Solo como pequeño matiz, cambiaría "we have chosen the bottom-up" por "we will use the bottom-up".